docs/user/archives.md

```diff
@@ -0,0 +1,59 @@
+# Archives
+
+Wheels and source distributions use the ZIP and tar format to distribute
+multiple files within a single artifact. To avoid archive differential / confusion
+attacks due to complexities of archive formats, PyPI rejects archives which
```

Comment:
question: is there any external links that describe more what this kind of attack is? I'm going to guess it'll be new for most folks reading this.